### ISSUE TITLE
Vuex getter triggers over again for each parallel call

### ISSUE THREAD

**Issue by dariuszsikorski** [vuejs/vuex#577]
I currently write some code compiler using Vuex, and return compiled code through vuex getter, like this:
```{{ $store.getters.getCompiledHtml }}```

The point is: I render ```compiledHtml``` directly in my project ```scene.vue``` and in ```output.vue```, both use same getter at the same time. The problem is the code gets compiled two times (once for each getter). How can I force getter to compile the code only once and return the same result in each parallel call?

**Comment by znck** [comment]
Provide minimal live reproduction using [https://jsfiddle.net](https://jsfiddle.net) or similar.

**Comment by LinusBorg** [comment]
> How can I force getter to compile the code only once and return the same result in each parallel call?

Getters work just like computed properties, in fact they *are* compued properties. so if the underlying data doesn't change, their result should be cached. 

If that's not the case for you, provide a repro, as znck said.

**Comment by ktsn** [comment]
Closing for inactivity. Please open a new issue if you can follow up further information.
